### PR TITLE
fix : Review fix java

### PR DESCRIPTION
**Pull request by sdkaarush**
## PR summary
Made changes in the repo after review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
-# Data Product Hub Services Java SDK Version 0.0.4
-Java client library to interact with various [Data Product Hub Services]().
+# Data Product Exchange Java SDK Version 0.0.4
+Java client library to interact with various [Data Product Exchange Services]().
```
Comment (padamstx) [inline README.md]:
```suggestion
Java client library to interact with various Data Product Exchange Service APIs.
```
Let's be as consistent as possible with the other SDKs.

File: README.md [hunk 2]
```diff
@@ -34,11 +34,11 @@ Changes might occur which impact applications that use this SDK.
 
 ## Overview
 
-The Data Product Hub Services Java SDK allows developers to programmatically interact with the following DPH services:
+The Data Product Exchange Services Java SDK allows developers to programmatically interact with the following DPH services:
```
Comment (padamstx) [inline README.md]:
```suggestion
The IBM Cloud Data Product Exchange Java SDK allows developers to programmatically interact with the following services:
```

File: README.md [hunk 3]
```diff
@@ -34,11 +34,11 @@ Changes might occur which impact applications that use this SDK.
...
 
 Service Name | Artifact Coordinates
 --- | ---
-[Data Product Hub Service](https://github.ibm.com/wdp-gov/data-product-api) | com.ibm.cloud:data-product-services:0.0.4
+[Data Product Exchange Services](https://test.cloud.ibm.com/apidocs/dataproducts) | com.ibm.cloud:dpx:0.0.4
```
Comment (padamstx) [inline README.md]:
```suggestion
[Data Product Exchange](https://cloud.ibm.com/apidocs/dataproducts) | com.ibm.cloud:dpx:0.0.4
```

**Review by padamstx — APPROVED** [review]
I suggested some minor changes, but will approve now to avoid a re-review.

File: README.md (hunk 1 above)
Comment (ShyamalaGowri) [reply README.md]: Updated as suggested

File: README.md (hunk 2 above)
Comment (ShyamalaGowri) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (ShyamalaGowri) [reply README.md]: Updated

**Pull request merged**